app/views/form_drafts/show.haml

```diff
@@ -13,13 +13,16 @@
   = form_tag controller: :forms, action: :submit, id: @draft.form.id do
     = fields_for :user do |u|
       .field
-        .heading Please complete the following personal information.
-        //.explanation You will be able to log in and review your submission.
+        .explanation 
+          This is the personal information we have for you.
```

Comment:
It might be clearest to explain what we will use this information for, since that's where the problem popped up.